### ISSUE TITLE
Update linux_repro_build_compare.sh to build just from the SBOM

### ISSUE THREAD

**Issue by andrew-m-leonard** [adoptium/temurin-build#3425]
With the update to add gcc, glibc and bootjdk to the SBOM: https://github.com/adoptium/temurin-build/pull/3385
The build from SBOM scripts on Linux: https://github.com/adoptium/temurin-build/blob/master/tooling/linux_repro_build_compare.sh
Needs updating to just use the SBOM.

**Comment by sophia-guo** [comment]
Also tried with Mac intel with docker got the same error.

**Comment by sophia-guo** [comment]
I noticed that for jdk21 jenkins build is using adoptopenjdk/centos7_build_image, trying centos7 see if there is still this problem

**Comment by sophia-guo** [comment]
Yes, change to centos7_build_image I can rebuild released jdk21 successfully.

**Comment by sophia-guo** [comment]
Diff ( host - mac x64, adoptopenjdk/centos7 )
JDK

```
./linux_repro_build_compare.sh https://github.com/adoptium/temurin21-binaries/releases/download/jdk-21.0.1%2B12/OpenJDK21U-sbom_x64_linux_hotspot_21.0.1_12.json https://github.com/adoptium/temurin21-binaries/releases/download/jdk-21.0.1%2B12/OpenJDK21U-jdk_x64_linux_hotspot_21.0.1_12.tar.gz
```


```
Binary files jdk-21.0.1+12/jmods/java.base.jmod and compare.31/jdk-21.0.1+12/jmods/java.base.jmod differ
diff -r jdk-21.0.1+12/lib/classlist compare.31/jdk-21.0.1+12/lib/classlist
355d354
< java/net/DefaultInterface
358,360d356
< java/net/Inet6Address
< java/net/Inet6Address$Inet6AddressHolder
< java/net/Inet6AddressImpl
366,367d361
< java/net/InterfaceAddress
< java/net/NetworkInterface
Binary files jdk-21.0.1+12/lib/modules and compare.31/jdk-21.0.1+12/lib/modules differ
Binary files jdk-21.0.1+12/lib/server/classes.jsa and compare.31/jdk-21.0.1+12/lib/server/classes.jsa differ
Binary files jdk-21.0.1+12/lib/server/classes_nocoops.jsa and compare.31/jdk-21.0.1+12/lib/server/classes_nocoops.jsa differ
diff -r jdk-21.0.1+12/release compare.31/jdk-21.0.1+12/release
12c12
< BUILD_SOURCE_REPO="https://github.com/adoptium/temurin-build.git"
---
> BUILD_SOURCE_REPO="https://github.com/adoptium/temurin-build"
```

**Comment by andrew-m-leonard** [comment]
@sophia-guo the ipv6 difference is a known problem: https://github.com/adoptium/temurin-build/issues/3429

**Comment by andrew-m-leonard** [comment]
> Yes, change to centos7_build_image I can rebuild released jdk21 successfully.

Yes, jdk17 and above require Centos7

**Comment by sophia-guo** [comment]
> @sophia-guo the ipv6 difference is a known problem: #3429 If you can, disable ipv6 in your VM ?

I'm using the adoptopenjdk/centos7_build_image

**Comment by andrew-m-leonard** [comment]
> > @sophia-guo the ipv6 difference is a known problem: #3429 If you can, disable ipv6 in your VM ?
> 
> I'm using the adoptopenjdk/centos7_build_image

ah ok, which host? it's probably the host that would need ipv6 disabling

**Comment by sophia-guo** [comment]
Closed by #3546
